qiskit/circuit/classical/expr/expr.py

```diff
@@ -151,8 +151,9 @@ def new(cls, name: str, type: types.Type) -> typing.Self:
 
     @property
     def standalone(self) -> bool:
-        """Whether this :class:`Var` is a standalone variable that owns its storage location.  If
-        false, this is a wrapper :class:`Var` around a pre-existing circuit object."""
+        """Whether this :class:`Var` is a standalone variable that owns its storage
+        location, if applicable. If false, this is a wrapper :class:`Var` around a
+        pre-existing circuit object."""
```

Comment:
is this just a line-wrapping change?  You might have an errant 80-character thing set somewhere haha

Reply:
I probably edited this and then "unedited" it, but left the line-wrap change. Sorry for the unnecessarily confusing diff here lol